dockers/docker-sonic-mgmt-framework/Dockerfile.j2

```diff
@@ -46,11 +32,11 @@ COPY ["start.sh", "rest-server.sh", "/usr/bin/"]
 COPY ["mgmt_vars.j2", "/usr/share/sonic/templates/"]
 COPY ["supervisord.conf", "/etc/supervisor/conf.d/"]
 
-# TODO: Remove this line once we no longer need Python 2
-RUN apt-get purge -y python-dev
-
 RUN apt-get remove -y g++ python3-dev
 RUN apt-get clean -y; apt-get autoclean -y; apt-get autoremove -y
 RUN rm -rf /debs
 
+RUN ln -sf /host_etc/passwd /etc/passwd
```

Comment:
This exposes /etc/passwd & /etc/group inside mgmt FW, which allows/handles Nortbound users' access.
Though this is conditioned upon, INCLUDE_MGMT_FRAMEWORK, this is concerning?

Should not this happen via d-bus ?

Reply:
Those links are not intended in my PR, added accidentally from other branch. It's removed now.

Reply:
This is discussed in the below thread too, but summarizing it here too:

Currently the /etc/passwd and /etc/group mounting is needed because the REST server is doing group lookups.
https://github.com/Azure/sonic-mgmt-framework/blob/master/rest/server/pamAuth.go#L71 :: IsAdminGroup()

We can eliminate the need to mount /etc/passwd and /etc/group in the container once the “AAA Improvements” (HAM) is merged to SONiC. See:
https://github.com/martin-belanger/SONiC/blob/aaa_improvements/doc/aaa/AAA%20Improvements/AAA%20Improvements.md#hamd-for-containers
Specifically, with the AAA improvements, the NSS module will take care of the user/group lookups and you don’t need to mount or link the /etc/passwd and /etc/group files anymore.

Since the AAA improvements have not been merged, we need those files accessible in the mgmt-framework. I don’t believe that’s a security concern, since you are not exposing anything additional to what a SONiC user already has access to.